pkgs/flox-manpages/default.nix

```diff
@@ -30,7 +32,7 @@ let
       --from markdown                          \
       --to man                                 \
        $source                                 \
-    > "$dest"
```

Comment:
It's a bit tangential but we might want to either read the section metadata from the md files themselves or pass the section metadata down at compile time here.
Non blocking.